cmd/kubeadm/app/constants/constants.go

```diff
@@ -107,11 +141,16 @@ const (
 	// DefaultEtcdVersion indicates the default etcd version that kubeadm uses
 	DefaultEtcdVersion = "3.0.17"
 
-	Etcd                  = "etcd"
-	KubeAPIServer         = "kube-apiserver"
+	// Etcd defines default etcd name
```

Comment:
These constants (Etcd, KubeAPIServer, KubeControllerManager, KubeScheduler, KubeProxy) are just used internally.
They are constants for referring to the component in question when for use in lists/maps/for loops etc.
Can you make a good comment of that?